source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -41,6 +41,17 @@ matching data is inserted.
 
 .. include:: /includes/access-cursor-note.rst
 
+Compatibility
+-------------
+
+.. |page-topic| replace:: use read operations
+.. |link-topic-ing| replace:: using read operations in the Atlas UI
+
+.. |atlas-url| replace:: :atlas:`Create, View, Update, and Delete Documents </atlas-ui/documents/#view--filter--and-sort-documents>`
```

Comment:
Q: Do you think it makes sense to use a different link title since this goes to the middle of the page?